src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -11,17 +11,37 @@ const ITEMS = [
     { label: 'Item 1', link },
     { label: 'Item 2', link },
     { label: 'Item 3', link },
+    { label: 'Item 4', link },
+    { label: 'Item 5', link },
 ];
 
 export default {
     title: 'Components/Breadcrumbs',
     component: Breadcrumbs,
     tags: ['autodocs'],
+    args: {
+        keepRoot: true,
+    },
     argTypes: {
         verticalGap: {
             options: Object.values(BreadcrumbGap),
             control: { type: 'radio' },
         },
+        truncate: {
+            description: 'If true, truncation will be applied to the trail.',
+            defaultValue: false,
+            control: { type: 'boolean' },
+        },
+        keepRoot: {
+            description: 'If true, the first item in the list will remain, else it is will truncated.',
+            defaultValue: true,
+            control: { type: 'boolean' },
+        },
+        includesCurrent: {
```

Comment:
What does it mean? that if false, the current could be within the overflow menu?
I see it's in the design but I'm wondering if it's right. I think the current should always be visible.
Maybe that's something that could be discussed in a Design Sync.

Reply:
It refers to if the current active crumbs should be rendered in line instead of below the trail.

Reply:
@imoutaharik I have updated the name to `activeInline` to attempt to be direct about its functionality

Reply:
Oh I see. thanks for the clarification.